test/store.js

```diff
@@ -0,0 +1,86 @@
+var test = require('tape')
+var Storage = require('../')
+var raf = require('random-access-file')
+var ram = require('random-access-memory')
+var path = require('path')
+var tmpdir = require('os').tmpdir
+var mkdirp = require('mkdirp')
+var { randomBytes } = require('hypercore-crypto')
+
+test('create without name', function (t) {
+  t.plan(2)
+  var s = Storage(ram)
+  var feed = s.create()
+  feed.ready(function () {
```

Comment:
This is an interesting little race condition. Technically you'd want

```js
var feed = s.create(function () {
  feed.ready(function () {
    // ...
  })
})
```

since either of these could finish first. I think it'd be good if `create()` didn't call its callback until `feed.ready()` has also fired, to simplify and reduce the possibilities of a race condition being hit.

Reply:
Hm, it's sort of awkward to use `feed` in a callback where it's being defined. Maybe `create` should *just* return the feed in its callback.. thoughts?